uint8 EYE_STATUS_NORMAL=0
uint8 EYE_STATUS_BLINK=1
uint8 EYE_STATUS_SURPRISED=2
uint8 EYE_STATUS_SLEEPY=3
uint8 EYE_STATUS_ANGRY=4
uint8 EYE_STATUS_SAD=5
uint8 EYE_STATUS_HAPPY=6
uint8 EYE_STATUS_TROUBLED=7
uint8 EYE_STATUS_DELIGHTED=8
uint8 EYE_STATUS_EXPECTING=9
uint8 EYE_STATUS_HEART=10
uint8 EYE_STATUS_SHINE=11
uint8 EYE_STATUS_FLUSTRATED=12
uint8 EYE_STATUS_BORING=13
